packages/client/README.md

```diff
@@ -0,0 +1,69 @@
+# `@lens-protocol/client`
+
+This package contains low level client for GraphQL Lens API.
+
+**Its interface will change without notice, use it at your own risk.**
+
+## Examples
```

Comment:
Good to keep the conversation going... but the proper place to document this will be on each API page as alternative to the current GQL examples.

Maybe over time we can make this leaner and reference the official docs.

Reply:
I know, I just wanted to keep it somewhere for now, till we start writing proper docs, it also helps to get a feeling for the SDK usage. We will remove it when sdk gets the proper shape.

Reply:
I like that we have some initial setup in the README showcasing the API, we should do the same for `@lens-protocol/react`